fastlane/Fastfile

```diff
@@ -29,9 +30,33 @@ platform :ios do
         run_tests
     end
 
+    desc "Creates SensitiveInfo.plist if it doesn't exist"
+    lane :create_sensitive_info do
+        if File.exists?("../Aux/SensitiveInfo.plist")
+            puts "SensitiveInfo.plist already available"
+        elsif
+            sh("cp -v '../Aux/SensitiveInfo-Sample.plist' '../Aux/SensitiveInfo.plist'")
+        end
+    end
+
+    desc "Populates SensitiveInfo.plist with environment values"
+    lane :populate_sensitive_info do |options|
+        appIdKey = "APPS_FLYER_APP_ID_%s" % options[:flavour]
```

Comment:
What happens if the flavour is Pre-Alpha or Beta?

Reply:
I guess you can find out by running Pre-Alpha build from your branch :)

Reply:
It leaves an empty value on the `APPS_FLYER_APP_ID`, which is fine since we don't care about AppsFlyer for those builds.

I can update the `AppsFlyerClient` to use `nilIfEmpty` on each key, but tbh I don't think there is any issue with starting the AppsFlyer SDK with invalid keys.